catalyst/callbacks/sklearn_classifier.py

```diff
@@ -0,0 +1,179 @@
+from typing import Callable
+
+from catalyst.core import CallbackOrder, IRunner
+from catalyst.core.callback import ICallback
+from catalyst.metrics._metric import AccumulationMetric
+
+
+class SklearnClassifierCallback(ICallback):

```

Comment:
🤔  btw, why it's not a `Callback`?

Reply:
I have no good reason to use `ICallback` here.
[Done](https://github.com/catalyst-team/catalyst/pull/1261/commits/f9ae449c3af3412a17c69ebb7c1b7e695966cb03)